Alignment/LaserAlignmentSimulation/src/LaserOpticalPhysicsList.cc

```diff
@@ -93,7 +93,6 @@ void LaserOpticalPhysicsList::ConstructProcess() {
   pManager->AddDiscreteProcess(theBoundaryProcess);
   pManager->AddDiscreteProcess(theWLSProcess);
 
-  theScintProcess->SetScintillationYieldFactor(1.);
```

Comment:
@civanch just for my understanding, is this removed due to SA warning? 
If yes, could you link it? thanks a lot!

Reply:
@francescobrivio , in recent Geant4 this value is 1. by default and in the next Geant4 11.0 this method is removed. I should extend the description of the PR.